### PR TITLE
Add python files for OOP session

### PR DESCRIPTION
**Pull request by ariefrahmansyah**

### CODE REVIEW

**Review by faisalmorensya — APPROVED** [review]
LGTM

**Review by ariefrahmansyah — COMMENTED** [review]
what have you done here? 😅

File: ariefrahmansyah/oop_encapsulation.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        self.__maxprice = price
+
+
+c = Computer()
+c.sell()
+
+# change the max price by changing the attribute directly
+# won't work!
+c.__maxprice = 1000
+c.sell()
```
Comment (imambungo) [inline ariefrahmansyah/oop_encapsulation.py:19]: if I add `print(c.__maxprice)` after `c.__maxprice = 1000`, it will print 1000. Why is that?

Comment (ariefrahmansyah) [reply ariefrahmansyah/oop_encapsulation.py:19]: Can you share your full code?

Comment (imambungo) [reply ariefrahmansyah/oop_encapsulation.py:19]: Here:

```python
class Computer:

    def __init__(self):
        self.__maxprice = 900

    def sell(self):
        print("Selling Price: {}".format(self.__maxprice))

    def setMaxPrice(self, price):
        self.__maxprice = price


c = Computer()
c.sell()

# change the max price by changing the attribute directly
# won't work!
c.__maxprice = 1000
print("Sellint Price: {}".format(c.__maxprice))
c.sell()

# using setter function
c.setMaxPrice(1000)
c.sell()
```

**Pull request merged**